pkg/pulsar/utils/namespace_name.go

```diff
@@ -83,7 +83,7 @@ func validateNamespaceName(tenant, namespace string) error {
 // and % is allowed as part of valid URL encoding
 const PATTEN = "^[-=:.\\w]*$"
 
-func checkName(name string) bool {
+func CheckName(name string) bool {
```

Comment:
Why did you expose this method?

Reply:
We are using pulsarctl in a terraform provider we writing, and exposed `CheckName` so that we could use it for validation in the terraform provider.

Reply:
@rivernate interesting to know that you are developing a terraform provider. Some of us also have the same idea (https://github.com/streamnative/terraform-provider-pulsar) and also use pulsarctl as the library. Is there something that we can collaborate on?